opm/output/eclipse/InteHEAD.hpp

```diff
@@ -43,6 +43,11 @@ namespace RestartIO {
             int maxWellsInField;
         };
 
+        struct WellDims {
+            int mxwlstprwel;
+            int mxdynwlst;
+        };
```

Comment:
Is there a reason to create a new structure here instead of adding the fields to the existing `wellTableDim` structure?

Reply:
Very good comment, I overlooked the WellTableDim struct! I have now deleted theWellDims struct and used WellTableDims instead, plus checked the tests - still OK! May be the WellTableDims could be renamed to WellDims to be more consistent with the keyword it represents in a future revision.

Reply:
> have now deleted theWellDims struct and used WellTableDims instead,

Thanks for the updates&mdash;this looks good to me.  I'll update the reference solutions to account for the new `INTEHEAD` items.

> May be the WellTableDims could be renamed to WellDims to be more consistent with the keyword it represents in a future revision.

Yes, I think that's a good idea.